src/workerd/io/actor-cache-test.c++

```diff
@@ -4655,18 +4825,24 @@ KJ_TEST("ActorCache transaction multiple counted delete batches") {
 
   mockStorage->expectCall("getMultiple", ws)
       // Note that this batch is smaller because "count2" was known to the actor cache.
-      .withParams(CAPNP(keys = ["count1"]), "stream"_kj)
+      .withParams(CAPNP(keys = ["count1", "count4"]), "stream"_kj)
       .useCallback("stream", [&](MockClient stream) {
     // Pretend that "count1" already exists but was not in the cache.
     stream.call("values", CAPNP(list = [(key = "count1", value = "1")]))
         .expectReturns(CAPNP(), ws);
     stream.call("end", CAPNP()).expectReturns(CAPNP(), ws);
-  }).expectCanceled();
+  }).thenReturn(CAPNP());
+
+  // TODO(now): I'm not fully sure I understand why this is a separate getMultiple?
```

Comment:
Isn't it a consequence of the `{.maxKeysPerRpc = 2}` config up at the start of this test? Or are you referencing something deeper than that?

Reply:
Thanks :) https://github.com/cloudflare/workerd/pull/1916/commits/6db7310a1d04ea381b6c0469970f9bb0b33b9208